pkg/gcp/error_handlers.go

```diff
@@ -8,23 +8,14 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-func (c *gcpClient) handleAttachImpersonatorError(err error) error {
+func (c *gcpClient) handleApiError(err error) error {
 	pApiError, ok := err.(*apierror.APIError)
 	if !ok {
 		return fmt.Errorf("Unexpected error")
```

Comment:
Not related to his MR: we should add some more context to this error - it's not helping as it is. I would assume it *should* never happen though.

Reply:
It should never happen. The call should return a google api error which the method is able to convert properly.